README.md

```diff
@@ -21,6 +21,26 @@ Link to the
 2. `cp config/airseeker.example.json config/airseeker.json` - To create the configuration file.
 3. `cp config/secrets.example.env config/secrets.env` - To create the secrets file.
 
+## Versioning and release
+
+Airseeker uses [semantic versioning](https://semver.org/). The version is specified in the `package.json` file. The
+package is not published to NPM, but instead dockerized and published to Docker Hub.
+
+To release a new version:
+
+1. `git checkout main` - Always version from `main` branch. Also, ensure that the working directory is clean (has no
+   uncommitted changes).
+2. `contracts:compile:force` - Build the latest Typechain artifacts.
+3. `pnpm version [major|minor|patch]` - Choose the right version bump. This will bump the version, create a git tag and
+   commit it.
+4. Build the docker image with tag `api3/signed-api:latest`. If running on Linux, use `pnpm run docker:build` otherwise
```

Comment:
This is a bit annoying, but I don't see anything that can be done if I want to do the releases from my machine. The proper solution would be releasing from CI.

Reply:
There is a WASM version of solc too, which is platform agnostic - it's a pity the tools don't support forcing use of that compiler. But as you noted building the contracts won't be needed once the package is published.